backend/routers/slack.router.js

```diff
@@ -24,85 +20,53 @@ if (process.env.NODE_ENV !== "test") {
 }
 
 //Finds Id number of channel
-router.get("/findId", (req, res) => {
-  publishMessage();
-  findEvent();
-  // findProject();
-});
+router.get("/scheduleMessage", findCurrentEvents, scheduleMessage);
```

Comment:
@Jonathanko52 In general, GET is for requesting any data, POST is for creating something. Please use POST for the endpoint because its purpose looks like "creating a new schedule for the message".
@nickbeaird The code review best practices recommend providing reason what caused the comment in case of opening a discussion. Or strict recommendation in case if you understand what is wrong.

Reply:
For both @alex-anakin  and @Jonathanko52 , I asked the question, so that I could ask some follow up questions. I feel like you have both a goal of having a GET route with the findCurrentEvents method, as well as a post with scheduleMessage. I think that both can be true. 

Also, for Alex, I believe that the rest of my comments were there to keep this one open ended question in context.